src/services/peopleSearch.ts

```diff
@@ -74,6 +74,29 @@ export type PeopleSearchQuery = {
   relationship_status?: string;
   involvement: string;
 };
+export interface SelectOption {
+  label: string;
+  value: string;
+}
```

Comment:
This interface is already defined in and exported from `views/PeopleSearch/components/SearchFieldList/components/SearchField`. Since that is the type that we are trying to produce, it's better to import it from there than to redefine it here and risk the two becoming inconsistent overtime.